### DISCUSSION TITLE
Support generated labels for loki datasource variables after formatting in label_vales

### DISCUSSION THREAD

**Discussion by mvkrishna86** [grafana/grafana#39618] [Dashboards, Panels & Visualizations]
The label_values() functionality while creating the variables is supporting only the existing labels of Loki. But a json or logfmt will create different labels based on the log format. It would be great to have the variables in Loki dashboards with the generated labels.
Something like this:
label_values({container="istio-proxy"} | json | __error__="",path!="", path)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
